snuba/consumer.py

```diff
@@ -86,27 +78,31 @@ def delivery_callback(self, error, message):
     def flush_batch(self, batch: Sequence[ProcessedMessage]):
```

Comment:
This method (`flush_batch`) is going to be removed at some point in the future and replaced with some sort of abstraction that  is generic on the input type to support streaming writes from the multiprocess consumer. I'm not sure what it'll look like yet but it will be quite a bit different than how this works today.